addons/sale_timesheet/__manifest__.py

```diff
@@ -43,8 +44,7 @@
         ],
         'web.assets_backend': [
             'sale_timesheet/static/src/scss/sale_timesheet.scss',
-            'sale_timesheet/static/src/js/project_overview.js',
-            'sale_timesheet/static/src/js/so_line_one2many.js',
+            'sale_timesheet/static/src/js/**/*',
```

Comment:
Okay for the moment, but it is not a good idea, because we could also have some JS files for assets_frontend in this path. 🙂

Reply:
Yeap, I thought about it.

Maybe when needed, the dev will define `frontend/` and `backend/` folders ?
If we always have to be specific, I don't see the gain of allowing us to use `*`, don't we agree ?

Reply:
Yeap, I'm agree with you but for this case, this project is not the `*` at the end, it is the `/**/*` this expression say we want to import files in all directory in `src/js/` but if a directory is for the assets_frontend is not good. 😬 

So, I think it is useful to use `*` when we are sure the directory will only be in assets_backend.
Thus, for your case, I think you are no choice to be specific.

Reply:
Indeed you're right !